_events/2021/2021-02-career-panel.md

```diff
@@ -0,0 +1,12 @@
+---
+title: "ADSA/US-RSE Early Career Panel"
+expires: 2021-02-23
```

Comment:
Did you mean to change the expires one to the 24th? Or is the event on the 24th?

Reply:
Lol! I goofed. I was attempting to do this and play bingo with my kids. Seems I lost at both. _Should_ be fixed. I'm going to go back to losing at bingo now...

Reply:
Thanks for checking carefully!